internal/distro/rhel85/partition_tables.go

```diff
@@ -5,230 +5,317 @@ import (
 	"math/rand"
 
 	"github.com/google/uuid"
+	"github.com/osbuild/osbuild-composer/internal/blueprint"
 	"github.com/osbuild/osbuild-composer/internal/disk"
 	"github.com/osbuild/osbuild-composer/internal/distro"
 )
 
-func defaultPartitionTable(imageOptions distro.ImageOptions, arch distro.Arch, rng *rand.Rand) disk.PartitionTable {
-	var sectorSize uint64 = 512
-	if arch.Name() == "x86_64" {
-		return disk.PartitionTable{
-			Size: imageOptions.Size,
-			UUID: "D209C89E-EA5E-4FBD-B161-B461CCE297E0",
-			Type: "gpt",
-			Partitions: []disk.Partition{
-				{
-					Bootable: true,
-					Size:     2048,
-					Start:    2048,
-					Type:     "21686148-6449-6E6F-744E-656564454649",
-					UUID:     "FAC7F1FB-3E8D-4137-A512-961DE09A5549",
-				},
-				{
-					Start: 4096,
-					Size:  204800,
-					Type:  "C12A7328-F81F-11D2-BA4B-00A0C93EC93B",
-					UUID:  "68B2905B-DF3E-4FB3-80FA-49D1E773AA33",
-					Filesystem: &disk.Filesystem{
-						Type:         "vfat",
-						UUID:         "7B77-95E7",
-						Mountpoint:   "/boot/efi",
-						FSTabOptions: "defaults,uid=0,gid=0,umask=077,shortname=winnt",
-						FSTabFreq:    0,
-						FSTabPassNo:  2,
-					},
-				},
-				{
-					Start: 208896,
-					Size:  imageOptions.Size/sectorSize - 208896 - 100,
-					Type:  "0FC63DAF-8483-4772-8E79-3D69D8477DE4",
-					UUID:  "6264D520-3FB9-423F-8AB8-7A0A8E3D3562",
-					Filesystem: &disk.Filesystem{
-						Type:         "xfs",
-						UUID:         uuid.Must(newRandomUUIDFromReader(rng)).String(),
-						Label:        "root",
-						Mountpoint:   "/",
-						FSTabOptions: "defaults",
-						FSTabFreq:    0,
-						FSTabPassNo:  0,
-					},
-				},
-			},
+const sectorSize = 512
+
+var x86PartitionTable = disk.PartitionTable{
+	UUID: "D209C89E-EA5E-4FBD-B161-B461CCE297E0",
+	Type: "gpt",
+	Partitions: []disk.Partition{
+		{
+			Bootable: true,
+			Size:     2048,
+			Start:    2048,
+			Type:     "21686148-6449-6E6F-744E-656564454649",
+			UUID:     "FAC7F1FB-3E8D-4137-A512-961DE09A5549",
+		},
+	},
+}
+
+var aarch64PartitionTable = disk.PartitionTable{
+	UUID:       "D209C89E-EA5E-4FBD-B161-B461CCE297E0",
+	Type:       "gpt",
+	Partitions: []disk.Partition{},
+}
+
+var ppc64lePartitionTable = disk.PartitionTable{
+	UUID: "0x14fc63d2",
+	Type: "dos",
+	Partitions: []disk.Partition{
+		{
+			Size:     8192,
+			Type:     "41",
+			Bootable: true,
```

Comment:
This changed in a recent PR and has a `Start` offset now.  Something to keep in mind when rebasing.